SU-HSDO/suhumsci
# SHS-5646): The multi-select box and single select box should have square edges in Traditional

# READY FOR REVIEW

## Summary
Make the multi-select box and single select box have square edges in Traditional

## Need Review By (Date)
06/19

## Urgency
medium

## Steps to Test
1. Go to the following pages
    - https://hs-traditional.suhumsci.loc/default-views/courses 
    - https://hs-colorful.suhumsci.loc/default-views/courses
2. Confirm that in Traditional, the multi-select box and single box have square edges.
3. Confirm that in Colorful, they are still rounded

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
